presentation-compiler/src/main/dotty/tools/pc/SelectionRangeProvider.scala

```diff
@@ -78,30 +82,38 @@ class SelectionRangeProvider(
   end selectionRange
 
   /** Given a tree, create a seq of [[SelectionRange]]s corresponding to that tree. */
-  private def selectionRangesFromTree(pos: SourcePosition)(tree: tpd.Tree)(using Context) =
+  private def selectionRangesFromTree(pos: SourcePosition)(tree: Tree)(using Context) =
     def toSelectionRange(srcPos: SourcePosition) =
       val selectionRange = new SelectionRange()
       selectionRange.setRange(srcPos.toLsp)
       selectionRange
 
     val treeSelectionRange = toSelectionRange(tree.sourcePos)
 
+    def getArgsSpan(args: List[Tree]): Option[SourcePosition] =
+      args match
+        case Seq(param) => Some(param.sourcePos)
```

Comment:
Is this needed? Won't it repeat the same selection? Can you add a test with a single param?

Reply:
@rochala do you need help finishing this one?